src/main/webapp/components/workspace/workspace.js

```diff
@@ -55,16 +60,27 @@ const workspaceById = gql`
   }
 `
 
-//TODO add paging
-const Results = ({ results }) =>
+const Results = ({ results, handleOpen, setResult }) =>
```

Comment:
Should we move the result index cards to their own file? The more interactions we add the larger this will get.

Reply:
I think that's a good idea. Probably better to do in a separate pr though

Reply:
It may also be worth decoupling result index cards from list view index cards